external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/DB2IntegrationSuite.scala

```diff
@@ -21,15 +21,11 @@ import java.math.BigDecimal
 import java.sql.{Connection, Date, Timestamp}
 import java.util.Properties
 
-import org.scalatest.Ignore
-
 import org.apache.spark.sql.Row
 import org.apache.spark.sql.types.{BooleanType, ByteType, ShortType, StructType}
 import org.apache.spark.tags.DockerTest
 
-
 @DockerTest
-@Ignore // AMPLab Jenkins needs to be updated before shared memory works on docker
```

Comment:
Not relevant change. Since docker tests are not integrated into jenkins we can turn this on.

Reply:
I'm seeing that other test suites for other DBMS don't have this, so good to remove to make it consistent.

Reply:
Not really following the discussion; are you guys saying this line should be removed? Because there's nothing changing here.

Reply:
Yes, as other tests don't have this. It's not a kind of "should be", but "can be".

Reply:
And it's removed in code diff as of now.

Reply:
I don't see it removed. It's still there.

Reply:
Sorry I should be more clearer - the change is removed, in other words, rolled back. No change.